spec/features/hiring_staff_can_sign_in_with_dfe_spec.rb

```diff
@@ -23,8 +23,7 @@
   scenario 'it does not sign-in the user, and tells the user what to do' do
     visit root_path
 
-    click_on(I18n.t('nav.sign_in'))
-    click_on(I18n.t('sign_in.link'))
+    sign_in_with_click
```

Comment:
Not sure what `sign_in_with_click` means. Could rename it to be something like `sign_in_user`?

Reply:
Agreed, will rename it

Reply:
renamed in https://github.com/DFE-Digital/teacher-vacancy-service/pull/862/commits/71561408199959b68ab9c785619b3f49cc244ca2